test/db/cmd/cmd_signature

```diff
@@ -202,11 +202,6 @@ EXPECT=<<EOF
 0x0044a030 142 flirt.sbrk
 0x0044a0c0 64 flirt.getpagesize
 0x0044a100 80 flirt.getdtablesize
-0x0044a2c0 327 flirt.trecurse
```

Comment:
did you check why these are removed?

Reply:
it is correct, the match was wrong. i checked manually